src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -0,0 +1,310 @@
+        No name translations
+      </v-flex>
+      <v-flex xs2 class="align-right" v-if="!hasNameTranslationChange">
+        <v-btn
+          id="correct-name-translation"
+          text color="primary"
+          @click="isEditing = true"
+        >
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Correct</span>
+        </v-btn>
+      </v-flex>
+      <v-flex xs2 class="align-right" v-else>
+        <v-btn
+          id="undo-name-translation"
+          text color="primary" class="undo-name-translation"
+          @click="resetNameTranslations"
+        >
+          <v-icon small>mdi-undo</v-icon>
+          <span>Undo</span>
+        </v-btn>
+
+        <!-- More Actions Menu -->
+        <span class="more-actions">
+          <v-menu offset-y>
+            <template v-slot:activator="{ on }">
+              <v-btn
+                text small color="primary"
+                class="more-actions-btn"
+                v-on="on"
+              >
+                <v-icon>mdi-menu-down</v-icon>
+              </v-btn>
+            </template>
+            <v-list>
+              <v-list-item
+                class="actions-dropdown_item"
+                @click="isEditing = true"
+              >
+                <v-list-item-subtitle>
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span class="ml-1">Correct</span>
+                </v-list-item-subtitle>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+        </span>
+      </v-flex>
+    </v-layout>
+    <v-layout v-else>
+      <v-flex xs3>
+        <label><strong>Name Translation(s)</strong></label>
+      </v-flex>
+      <v-flex xs9>
+        <v-layout>
+          <v-flex>
+            <p>Name translations must use the Latin Alphabet (English, French, etc.).
+              Names that use other writing systems must spell the name phonetically in English or French.
+            </p>
+            <v-btn outlined color="primary" @click="isAddingNameTranslation=true" :disabled="isAddingNameTranslation">
+              <v-icon>mdi-plus</v-icon>
+              <span>Add Name Translation</span>
+            </v-btn>
+          </v-flex>
+        </v-layout>
+        <v-layout>
+          <v-flex>
+            <add-name-translation
+              v-if="isAddingNameTranslation"
+              :editNameTranslation="editingNameTranslation"
+              @addTranslation="addName($event)"
+              @cancelTranslation="cancelOrResetEditing()"
+            ></add-name-translation>
+            <list-name-translation
+              v-if="drafTranslations && drafTranslations.length > 0"
+              :isAddingNameTranslation="isAddingNameTranslation"
+              :translationList="drafTranslations"
+              @editNameTranslation="editNameTranslation($event)"
+              @removeNameTranslation="removeNameTranslation($event)"
+            />
+          </v-flex>
+        </v-layout>
+        <v-layout pt-5>
```

Comment:
Does this prop actually work, or should this be a class value?

Reply:
Yes it does

Reply:
OK. I just learned something!